frontend/webapp/render/views/main_view.py

```diff
@@ -25,7 +25,7 @@ def display_monitors():
     )
 
 
-@main.route("/hijacks/")
+@main.route("/hijacks", methods=["GET"])
```

Comment:
this is a small fix so that hijack updates with multiple hijack keys can be properly displayed